pkg/cmd/run.go

```diff
@@ -80,7 +80,7 @@ func run(_ *cobra.Command, _ []string) {
 					err = tmpl.Execute(w, struct {
 						HTTPServerURL string
 					}{
-						HTTPServerURL: fmt.Sprintf("localhost:%d", config.HTTPPort),
+						HTTPServerURL: config.HTTPAddr,
```

Comment:
todo: need to fix this @jon-whit . The default `:8080` doesn't work here obviously, so we'll need to account for that.

Reply:
Alright, I resolved this by waiting for a tcp connection to be established to the HTTP server and then dynamically grabbing the remote host address from the connection. So makes it work regardless of what host:port address you wire the HTTP server up with 👍